custom_components/renault/strings.json

```diff
@@ -4,7 +4,6 @@
       "kamereon_no_account": "Unable to find Kamereon account."
```

Comment:
From a general point of view, this file is useless in a custom component. Do we need to keep it here even if we plan to merge in Home Assistant core?

Reply:
I think it is better to keep it for the future migration.

Reply:
OK